lib/algolia_index/base.rb

```diff
@@ -0,0 +1,61 @@
+module AlgoliaIndex
```

Comment:
Minor one, but perhaps this should be in `alogolia_index/models`, given it's the model base class?